airbyte-webapp/src/views/Connection/ConnectionForm/ConnectionFormFields.tsx

```diff
@@ -0,0 +1,184 @@
+import { faSyncAlt } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { Field, FieldProps, Form } from "formik";
+import React from "react";
+import { FormattedMessage, useIntl } from "react-intl";
+import styled from "styled-components";
+
+import { Button, Card, ControlLabels, H5, Input } from "components";
+
+import { NamespaceDefinitionType } from "core/request/AirbyteClient";
+import { useConnectionFormService } from "hooks/services/Connection/ConnectionFormService";
+import { ValuesProps } from "hooks/services/useConnectionHook";
+
+import { NamespaceDefinitionField } from "./components/NamespaceDefinitionField";
+import ScheduleField from "./components/ScheduleField";
+import SchemaField from "./components/SyncCatalogField";
+import { FormikConnectionFormValues } from "./formConfig";
+
+interface SectionProps {
+  title?: React.ReactNode;
+}
+
+const TryArrow = styled(FontAwesomeIcon)`
+  margin: 0 10px -1px 0;
+  font-size: 14px;
+`;
+
+export const StyledSection = styled.div`
+  padding: 20px 20px;
+  display: flex;
+  flex-direction: column;
+  gap: 15px;
+
+  &:not(:last-child) {
+    box-shadow: 0 1px 0 rgba(139, 139, 160, 0.25);
+  }
+`;
+
+export const FlexRow = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: flex-start;
+  gap: 10px;
+`;
+
+export const LeftFieldCol = styled.div`
+  flex: 1;
+  max-width: 640px;
+  padding-right: 30px;
+`;
+
+export const RightFieldCol = styled.div`
+  flex: 1;
+  max-width: 300px;
+`;
+
+export const LabelHeading = styled(H5)`
+  line-height: 16px;
+  display: inline;
+`;
+
+export const ConnectorLabel = styled(ControlLabels)`
+  max-width: 328px;
+  margin-right: 20px;
+  vertical-align: top;
+`;
+
+const NamespaceFormatLabel = styled(ControlLabels)`
+  flex: 5 0 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+`;
+
+export const FormContainer = styled(Form)`
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+`;
+
+export const Section: React.FC<React.PropsWithChildren<SectionProps>> = ({ title, children }) => (
```

Comment:
While we're thinking about reorganizing folder structure -- I feel like there's a convo to be had about declaring multiple components within a file like this (I know this was a copy/paste from what existed before, but we have this pattern in a few spots)

Reply:
Moved!